system_core/tests/common.py

```diff
@@ -0,0 +1,76 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import psutil
+
+from datadog_checks.dev import get_here
+from datadog_checks.utils.platform import Platform
+
+HERE = get_here()
+CHECK_NAME = "system_core"
+
+INSTANCE = {
+    "integ": {}
```

Comment:
Let's do a config with `tags` in it, since it appears to be the only key expected by the check